docs/docs/user/examples/atm-fraud-pipeline.md

```diff
@@ -1,9 +1,9 @@
 # ATM fraud detection pipeline
 
-ATM fraud is a demo pipeline for ATM fraud detection. 
-The original by Confluent is written in KSQL 
-and outlined in this [blogpost](https://www.confluent.io/blog/atm-fraud-detection-apache-kafka-ksql/). 
-The one used in this example is re-built from scratch using [bakdata](https://bakdata.com/)'s 
+ATM fraud is a demo pipeline for ATM fraud detection.
```

Comment:
I don't see the point of the documentation changes in this PR please remove these changes from this PR.